pkg/oc/bootstrap/docker/run_self_hosted.go

```diff
@@ -270,6 +253,20 @@ func (c *ClusterUpConfig) BuildConfig() (*configDirs, error) {
 		}
 	}
 
+	substitutions := map[string]string{
+		"/path/to/master/config-dir":              configLocations.masterConfigDir,
+		"/path/to/openshift-apiserver/config-dir": configLocations.openshiftAPIServerConfigDir,
+		"ETCD_VOLUME":                             "emptyDir:\n",
+	}
+	if len(c.HostDataDir) > 0 {
+		substitutions["ETCD_VOLUME"] = `hostPath:
+      path: ` + c.HostDataDir + "\n"
+	}
+	for _, staticPodLocation := range staticPodLocations {
```

Comment:
you dropped the log statement here, maybe intentionally

Reply:
unintentionally, good catch, added it back.